src/Bundler.js

```diff
@@ -87,16 +87,21 @@ class Bundler extends EventEmitter {
     this.packagers.add(type, packager);
   }
 
-  loadPlugins() {
+  loadPlugins(location = this.mainFile) {
     try {
-      let pkg = localRequire('./package.json', this.mainFile);
+      let pkg = localRequire('./package.json', location);
       let deps = Object.assign({}, pkg.dependencies, pkg.devDependencies);
       for (let dep in deps) {
         if (dep.startsWith('parcel-plugin-')) {
-          localRequire(dep, this.mainFile)(this);
+          localRequire(dep, location)(this);
         }
       }
     } catch (err) {
```

Comment:
Is `err` always because it isn't found, or could it be something like access denied? Wouldn't want to keep traversing if it is found but just couldn't read it. That's a legitimate error.

Reply:
I'll look into it